hypervisor/include/arch/x86/lapic.h

```diff
@@ -51,17 +51,54 @@ enum intr_cpu_startup_shorthand {
 	INTR_CPU_STARTUP_UNKNOWN,
 };
 
+
+/**
+ * @brief Save context of lapic
+ *
+ * @param[inout]	regs	Pointer to struct lapic_regs to hold the
+ *				context of current lapic
+ */
 void save_lapic(struct lapic_regs *regs);
 void early_init_lapic(void);
 void init_lapic(uint16_t pcpu_id);
 void send_lapic_eoi(void);
+
+/**
+ * @brief Get the lapic id
+ *
+ * @return lapic id
+ */
 uint32_t get_cur_lapic_id(void);
+
+/**
+ * @brief Send an SIPI to a specific cpu
+ *
+ * Send an Startup IPI to a specific cpu, to notify the cpu to start booting.
+ *
+ * @param[in]	intr_cpu_startup_shorthand The startup_shorthand 
```

Comment:
You have a mismatch here between `intr_cpu_startup_shorthand` and `cpu_startup_shorthand`
```suggestion
 * @param[in]	cpu_startup_shorthand The startup_shorthand
```